# Describes a variable and its value related to the error
#
# Part of the state message

# Type of reference (eg headerId, orderId, actionId, etc)
#
# Field type: Required
string reference_key

# Value of the the key
#
# Field type: Required
string reference_value
